style.css

```
@@ -0,0 +1,113 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.about_us{
+    padding: 40px 0px;
+    background-color: #dff4f5;
+    height: 840px;  
+    
+}
+.container{
+    width: 100%;
+    padding-right:15px ;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+    
+}
+
+.About_A{
+    text-align: center;
+    padding-bottom:50px ;
+    margin-top: 40px;
+    width: 960px;
+    height: 186px;
+}
```

update this style as the following : 
```suggestion
.About_A{
    text-align: center;
    width: 100%;
    height: 27vh;
}
```